ui/lib/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { Dayjs } from 'dayjs'
```

Comment:
I think you also need to expose it in `components/index.ts`.

Reply:
Added.